src/app/client/client-info/client-info.component.spec.ts

```
@@ -0,0 +1,72 @@
+      imports: [ClientInfoComponent, NoopAnimationsModule],
+      providers: [provideHttpClientTesting(), provideHttpClient()],
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(ClientInfoComponent);
+    component = fixture.componentInstance;
+    clientService = TestBed.inject(ClientService);
+    httpMock = TestBed.inject(HttpTestingController);
+    clientService['clientDataSubject'].next(mockClientData);
+  });
```

Biome

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)